msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -945,26 +959,39 @@ internal void ClearCache()
         {
             lock (LockObject)
             {
-                try
+                ClearMsalCache();
+                ClearAdalCache();
+            }
+        }
+
+        internal void ClearAdalCache()
+        {
+            IDictionary<AdalTokenCacheKey, AdalResultWrapper> dictionary = AdalCacheOperations.Deserialize(legacyCachePersistance.LoadCache());
```

Comment:
In adal version there are log messages here...can you add them here for msal (or move to core instead).